docs/tutorial.rst

```diff
@@ -742,50 +742,83 @@ Working with Metadata
 Metadata is information associated with entities that tskit doesn't use or interpret,
 but which is useful to pass on to downstream analysis such as sample ids, dates etc. See
 :ref:`sec_metadata` for a full discussion. Each table has a
-``metadata_schema`` which details the
+:class:`tskit.MetadataSchema` accessed as ``table.metadata_schema``. which details the
 contents and encoding of the metadata for each row. A metadata schema is a JSON document
 that conforms to `JSON Schema <https://json-schema.org/understanding-json-schema/>`_
 (The full schema for tskit is at :ref:`sec_metadata_schema_schema`).
-Here's a simple example schema, which we'll apply to the individuals table:
+
+++++++++++++++++++++++++++++
+Reading metadata and schemas
+++++++++++++++++++++++++++++
+
+Metadata is automatically decoded using the schema when accessed via a
+:class:`TreeSequence` or :class:`TableCollection` Python API. For example:
+
+.. code-block:: python
+
+    >>> ts.individual(0).metadata
+    {'accession': 'ERS0001', 'pcr': True}
+    >>> ts.tables.individuals[0].metadata
+    {'accession': 'ERS0001', 'pcr': True}
+
+Viewing the :class:`tskit.MetadataSchema` for a table can help with understanding
+its metadata as it can contain descriptions and constraints:
 
 .. code-block:: python
 
-    raw_schema = {
-        "codec": "json",
-        "type": "object",
-        "properties": {
-            "accession": {"type": "string"},
-            "pcr": {
-                "name": "PCR Used",
-                "description": "Was PCR used on this sample",
-                "type": "boolean",
-            },
+    >>> print(ts.table_metadata_schemas.individual)
+    {
+      "codec": "json",
+      "type": "object",
+      "properties": {
+        "accession": {
+          "type": "string"
+          "description": "ENA accession number"
         },
-        "required": ["accession", "pcr"],
-        "additionalProperties": False,
+        "pcr": {
+          "name": "PCR Used",
+          "description": "Was PCR used on this sample",
+          "type": "boolean"
+        }
+      },
+      "required": [
+        "accession",
+        "pcr"
+      ],
+      "additionalProperties": false
     }
+    >>> print(ts.tables.individuals.metadata_schema)
+    (.... as above)
+
+The top-level metadata and schemas for the entire tree sequence is similarly
+accessed with ``ts.metadata`` and ``ts.metadata_schema``.
+
+++++++++++++++++++++++++++++++
+Modifying metadata and schemas
+++++++++++++++++++++++++++++++
 
-The ``codec`` entry in the schema specifies how the metadata should be stored.
-See :ref:`sec_metadata_codecs` for details on the supported codecs. This schema
-defines two properties, both of which are mandatory as they are in the ``required``
-list. To avoid errors we set ``additionalProperties`` to false, as this causes any
-unexpected properties in the metadata to fail validation.
+If you are creating or modifying a tree sequence you may want to record or add to the
```

Comment:
This paragraph isn't really parsing for me. What are you trying to say here?

Reply:
Tried to make it clearer in https://github.com/tskit-dev/tskit/pull/741/commits/10ba7fff04572c58e663402f2a7ce65242464f6c

Reply:
Ah, that's better, now I get it!